low-level/json_msgs/messages/sensors/cpu_data.py

```diff
@@ -60,6 +60,7 @@ def __init__(self, host_id,
                        node_id,
                        cluster_id,
                        alert_type,
+                       event,

```

Comment:
what would be event string here? This is what shows up as description in GUI rt?